native/cocos/renderer/gfx-empty/EmptyDevice.cpp

```diff
@@ -73,7 +73,6 @@ bool EmptyDevice::doInit(const DeviceInfo & /*info*/) {
 
     _formatFeatures.fill(static_cast<FormatFeature>(-1)); // allow all usages for all formats
     _formatFeatures[toNumber(Format::UNKNOWN)] = FormatFeature::NONE;
-    _formatFeatures[toNumber(Format::COUNT)] = FormatFeature::NONE;
```

Comment:
Why delete it?

Reply:
verctor define:
```
std::array<FormatFeature, static_cast<size_t>(Format::COUNT)> _formatFeatures;
```
the assign is out fo range.